override-dev-local.yaml

```diff
@@ -3,7 +3,7 @@ osbs:
             name: containers/redhat-sso-7
             branch: jb-sso-cd-dev-rhel-7
 artifacts:
-      - sha256: d8f7733cad3f8cc0daa14bfd70b1c6e1b116744baef67fd71bb02249e590f8f2
+      - md5: c6c5690dd226280b612eed960d66a2f9
```

Comment:
I think we don't have change override-dev-local.yaml. This is used for local builds only I think.

Reply:
The old artefact was for EAP CD15, so we need to change it.